src/components/BigButton.tsx

```diff
@@ -4,9 +4,9 @@ import COLORS from '@/styles/colors';
 const BigButton = styled.button<{ $color?: string }>`
   background-color: ${props => props.color || COLORS.blueMid};
   color: ${props => props.$color};
-  padding: 0.94rem 11.5rem;
+  padding: 0.94rem 0rem;
```

Comment:
minor note: for values like 0, we don't need to include the units, so just `0` is fine, but it is up to personal preference!

Reply:
gotcha! changed